src/react/projects/spark-core-react/src/SprkCard/components/SprkCardHighlightedHeader/SprkCardHighlightedHeader.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SprkCardHighlightedHeader = (props) => {
+  const { highlightedHeaderConfig: { description, title, bodyText } } = props;
+
+  return (
+    <React.Fragment>
+      <div className="sprk-o-Stack__item sprk-c-Card__header sprk-o-Stack sprk-o-Stack--medium">
+        <h3 className="sprk-b-TypeDisplaySeven sprk-o-Stack__item sprk-u-Color--white">
+          {description}
+        </h3>
+        <h4 className="sprk-b-TypeDisplayFive sprk-o-Stack__item sprk-u-Color--white">{title}</h4>
+      </div>
+      <div className="sprk-o-Stack__item sprk-o-Stack sprk-o-Stack--medium sprk-c-Card__content">
+        <p className="sprk-b-TypeBodyTwo sprk-o-Stack__item">{bodyText}</p>
+      </div>
+    </React.Fragment>
+  );
+};
+
+SprkCardHighlightedHeader.propTypes = {
+  highlightedHeaderConfig: PropTypes.shape({
```

Comment:
Should highlightedHeaderConfig be required here?

Reply:
Short answer: no
If someone want to use highlighted header, the proptype of highlightedHeaderConfig needs to be passed in so there's no need to require it. If we did decide to do this, we wouldn't be able to let some things be optional. This becomes increasingly problematic with teaser config if we were to follow this pattern. Since required proptypes don't need default proptypes, we would take away defaults like links and mediaVariants.